src/core/api.proto

```diff
@@ -223,6 +223,25 @@ message InferRequestHeader
   //@@     request.
   //@@
   repeated Output output = 3;
+
+  //@@  .. cpp:var:: uint32 priority [DLIS-1083]
+  //@@
+  //@@     The priority value of this request. If the priority value is not
```

Comment:
"If priority handling is not enabled for the model, then this value is ignored."